docs-src/pages/UiAutocomplete.vue

```diff
@@ -151,7 +151,7 @@
                                 <td></td>
```

Comment:
Can't comment on unchanged line above, but the `value` prop should now be `modelValue`.